docs/versioning.rst

```diff
@@ -0,0 +1,37 @@
+.. _kinto-versioning:
+
+Versioning
+##########
+
+
+Kinto implements an :term:`HTTP API` that can be consumed by clients like
+`Kinto.js <http://kintojs.readthedocs.org/>`_.
+
+The :term:`HTTP API` is described :ref:`here <kinto-api-endpoints>` and is
+subject to changes. It follows the :term:`Cliquet Protocol`.
+
+When the :term:`HTTP API` is changed, its version is incremented.
+The :term:`HTTP API` version follows a :term:`Semantic Versioning`
+pattern and uses this rule to be incremented:
+
+1. any change to the :term:`HTTP API` that is backward compatible increments
+   the **MINOR** number, and the modification in the documentation should reflect
+   this with a header like "Added in 1.x".
+
+2. any change to the :term:`HTTP API` that is backward incompatible increments
+   the **MAJOR** number, and the differences are summarized at the begining of
+   the documentation, a new document for that **MAJOR** version is created.
+
+We want to avoid **MAJOR** changes as much as possible in the future, and stick
+with 1.x as long as we can.
+
+A client that interacts with a Kinto server can query the server to know what
+is its :term:`HTTP API` version. This is done with a query on the root view,
+as described in :ref:`the root API description <api-utilities>`.
```

Comment:
I'd add *it's up to clients to ensure they fully support the HTTP API version provided by the server*